### PR TITLE
LPS-118832 SF

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Some context in liferay-frontend#444

TODOS:

- [x] Remove componentId
- [x] Use document.getElementById
- [x] Remove servletContext
- [x] Use Liferay.componentRead instead of _searchContainer
- [x] Add DisplayContext
- [x] Make a Class with dispose method?

### CODE REVIEW

File: modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/display/context/EditMBDisplayContext.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+import javax.portlet.PortletURL;
+import javax.portlet.ResourceURL;
+
+/**
+ * @author Ambrín Chaudhary
+ */
+public class EditMBDisplayContext {
+
+	public EditMBDisplayContext(
```
Comment (4lejandrito) [inline modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/display/context/EditMBDisplayContext.java:37]: Rename it to `MBEditMessageDisplayContext` to keep it consistent with the others.

File: modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/edit_message.jsp [hunk 2]
```diff
@@ -460,43 +460,10 @@ if (portletTitleBasedNavigation) {
...
-	<%
-	taglibContext.put("getAttachmentsURL", getAttachmentsURL);
-	taglibContext.put("viewTrashAttachmentsURL", viewTrashAttachmentsURL);
-	%>
-
-</c:if>
-
 <liferay-frontend:component
-	context="<%= taglibContext %>"
+	context="<%= editMBDisplayContext.getComponentContext() %>"
```
Comment (4lejandrito) [inline modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards/edit_message.jsp:467]: Maybe rename it to `getMBPortletComponentContext`?

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by ambrinchaudhary** [comment]
Commits added to the original pull in FI: https://github.com/liferay-frontend/liferay-portal/pull/444

Thanks @4lejandrito for your review.

**Pull request closed without merging**